docs/chapters/03-3.md

```diff
@@ -26,22 +26,23 @@ If our data exhibits stationarity, locality, and compositionality, we can exploi
+<img src="Figure 1 Fully Connected Network.jpg" width="350px" /><br>
+*Figure 1: Fully Connected Network*
 
-| <img src="https://i.imgur.com/RX2bhht.png" width="300"/> | <img src="https://i.imgur.com/mJTCMyZ.png" width="300"/>|
-|*Figure 2(a): Before Applying Sparsity* | *Figure 2(b): After Applying Sparsity*|
+If our data exhibits locality, each neuron needs to be connected to only a few local neurons of the previous layer. Thus, some connections can be dropped as shown in Fig.2. Fig.2(a) represents an FC network. Taking advantage of the locality property of our data, we drop connections between far away neurons in Fig.2(b). Although the hidden layer neurons (green) in Fig.2(b) don't span the whole input, the overall architecture will be able to account for all input neurons. The receptive field (RF) is the number of neurons of previous layers, that each neuron of a particular layer can see or has taken into account. Therefore, the RF of the output layer w.r.t the hidden layer is 3, RF of the hidden layer w.r.t the input layer is 3, but the RF of the output layer w.r.t the input layer is 5. 
 
+<img src="Figure 2(a) Before Applying Sparsity.png" width="300"/>  <img src="Figure 2(b) After Applying Sparsity.png" width="300"/> <br>
+*Figure 2(a): Before Applying Sparsity* &nbsp; &nbsp; &nbsp; *Figure 2(b): After Applying Sparsity*
 
 ### Stationarity $\Rightarrow$ parameters sharing
 
 If our data exhibits stationarity, we could use a small set of parameters multiple times across the network architecture. For example in our sparse network, Fig.3(a), we can use a set of 3 shared parameters (yellow, orange and red). The number of parameters will then drop from 9 to 3! The new architecture might even work better because we have more data for training those specific weights.
 The weights after applying sparsity and parameter sharing is called a convolution kernel.
 
-|<img src="https://i.imgur.com/E2w4QoY.png" width="300"/> | <img src="https://i.imgur.com/l08MOj9.png" width="300"/>|
-|*Figure 3(a): Before Applying Parameter Sharing* | *Figure 3(b): After Applying Parameter Sharing*|
+<img src="Figure 3(a) Before Applying Parameter Sharing.png" width="300"/> <img src="Figure 3(b) After Applying Parameter Sharing.png" width="300"/> <br>
+ *Figure 3(a): Before Applying Parameter Sharing* &nbsp; &nbsp; &nbsp; &nbsp; *Figure 3(b): After Applying Parameter Sharing*
+
```

Comment:
I've added a table here, in order to display the figures side by side.
Please, don't remove what I've done.